elasticdl/python/common/model_handler.py

```diff
@@ -7,10 +7,12 @@
 from elasticdl.python.common.log_utils import default_logger as logger
 from elasticdl.python.common.save_utils import CheckpointSaver
 from elasticdl.python.elasticdl.layers.embedding import Embedding
+from elasticdl.python.keras.layers import SparseEmbedding
```

Comment:
The difference between `elasticdl.python.elasticdl.layers` and  `elasticdl.python.keras.layers `?

Reply:
I create a `elasticdl.python.keras.layers` directory to place the layer to show that the layer is Keras-native and can be used by users to define their models. I think the layers in `difference between elasticdl.python.elasticdl.layers ` will not be exposed.